docs/AOT.md

```diff
@@ -163,7 +163,7 @@ The makefile provides the following targets:
 * `download_weights`: it downloads the Resnet50 network model in the Caffe2 format.
 * `build/resnet50.o`: it generates the bundle files using the Glow image-classifier as described above.
   The concrete command line looks like this:
-  `image-classifier tests/images/imagenet/cat_285.png -image_mode=0to1 -m resnet50 -cpu -emit-bundle build`
+  `image-classifier tests/images/imagenet/cat_285.png -image_mode=0to1 -m resnet50 -model_input_name=gpu_0/data -cpu -emit-bundle build`
```

Comment:
nit: -m resnet50 =? -m=resnet50 for consistency.

Reply:
Done